source/_integrations/plugwise.markdown

```diff
@@ -156,48 +144,43 @@ action: `climate.turn_off`, `climate.turn_on` (Adam only)
 These actions will switch the Adam regulation mode (= HVAC system mode) to off or on, affecting the operation of all connected thermostats.
 `climate.turn_on` will activate the previously selected heating or cooling mode.
 
-Example:
-
-```yaml
-# Example script climate.turn_off
-script:
-  turn_heating_on:
-    sequence:
-      - action: climate.turn_off
-        target:
-          entity_id: climate.bios
-```
-
-#### Change climate schedule
-
-action: `select.select_option`
+Automation example:
 
 ```yaml
-# Example script change the thermostat schedule
-script:
-  lisa_change_schedule:
-    sequence:
-      - action: select.select_option
-        target:
-          entity_id: select.bios_thermostat_schedule
-        data:
-          option: "Regulier"
+# Example automation using climate.turn_off for your cinema room
+alias: Stop climate in cinema after movie
+description: "Automatically turn off climate when the movie ends."
+triggers:
+  - trigger: state
+    entity_id:
+      - media_player.cinema
+    attribute: sound_mode
+    from: Movie
+condition:
+  - condition: state
+    entity_id: climate.cinema
+    state: 'on'
+actions:
+  - action: homeassistant.turn_off
+    target:
+      entity_id: climate.cinema
+  - delay:
+      seconds: 30
+mode: single
 ```
 
-#### Change boiler setpoint
+#### Update gateway data
 
-action: `number.set_value`
+Forced update of data from your gateway can be triggered by calling the generic `homeassistant.update_entity` action.
 
```

Comment:
Why would we add this? In general, users shouldn't be or need to use this at all?